src/__tests__/hooks/useDragAndDropLanguagesHook.test.ts

```diff
@@ -0,0 +1,84 @@
+import { act, renderHook } from "@testing-library/react";
+import { useDragAndDropLanguagesHook } from "../../hooks/useDragAndDropLanguagesHook";
+
+const mockLanguages = [
+  {
+    id: "d108e5aa-e058-4245-8a9a-e056a1594dfa",
+    name: "Francès",
+    level: "Intermedi"
+  },
+  {
+    id: "d108e5aa-e058-4245-8a9a-e056a1594dfg",
+    name: "Anglais",
+    level: "Avançat",
+  }
+]
+
+describe("UseDragAndDropLanguagesHook", () => {
+
+  it("should render the App component", () => {
```

Comment:
The assertion name dont have relation with the test

Reply:
Corregido